src/router/kromo/dd-wrt/lang_pack/german.js

```diff
@@ -1432,7 +1432,7 @@ routetbl.not="Nicht";
 routetbl.from="Von";
 routetbl.to="Nach";
 routetbl.priority="Priorität";
-routetbl.tos="Tos/Dsfield";
+routetbl.tos="TOS/DS Field";
```

Comment:
The right one here is `ToS/DS Field` doesnt matter anyway. I have a patch incommingfor english strings as well

Reply:
Saw it and decided to leave it as TOS.

Some things you should not take so exactly.
in the german wiki it is also written TOS

https://de.wikipedia.org/wiki/IP-Paket#TOS_(Type_of_Service)

Reply:
Yea, translations have latitude to do whatever, English strings not so much

Reply:
But if you others find ToS better I can also change all the strings (is not only one and a work of few minutes)

Reply:
see below